packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -19,7 +19,6 @@ import type { AgentConfigOptions as RUMAgentConfigOptions } from '@elastic/apm-r
 const DEFAULT_CONFIG: AgentConfigOptions = {
   active: false,
   environment: 'development',
-  logUncaughtExceptions: true,
```

Comment:
Self review: It seems this config was removed in 4.x and now uncaughtExceptions are captured by default

Reply:
https://www.elastic.co/guide/en/apm/agent/nodejs/current/configuration.html#capture-exceptions

Reply:
https://github.com/elastic/apm-agent-nodejs/blob/main/docs/upgrade-to-v4.asciidoc#loguncaughtexceptions